velox/functions/sparksql/JsonObjectKeys.h

```diff
@@ -15,6 +15,7 @@
  */
 #pragma once
 
+#include "velox/functions/Macros.h"
```

Comment:
Why do we need this change?

Reply:
for `VELOX_DEFINE_FUNCTION_TYPES`

Reply:
Still not understand why we do not need it before this PR...

Reply:
but it needed now.